worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/IngestionBatchWorkerPlugin.java

```diff
@@ -88,6 +92,20 @@ public IngestionBatchWorkerPlugin()
         simpleModule.addDeserializer(DocumentWorkerDocument.class,
                                      new DocumentWorkerDocumentDeserializer(totalSubdocumentLimit.get()));
         mapper.registerModule(simpleModule);
+
+        validationFile = System.getenv("CAF_VALIDATION_FILE");
+        if (StringUtils.isNotEmpty(validationFile)) {
+            try {
+                fieldValidator = new FieldValidator(validationFile);
+            } catch (final ValidationFileAdapterException validationFileAdapterException) {
+                log.error("Exception when attempting to read validation file" + "\n"
+                    + validationFileAdapterException.getMessage());
+                throw new RuntimeException("Exception when attempting to read validation file" + "\n"
+                    + validationFileAdapterException.getMessage());
```

Comment:
Generally I would avoid putting newlines in exception messages.  Also, when re-throwing an exception, pass the original exception up as the cause:
```suggestion
                throw new RuntimeException("Exception when attempting to read validation file: "
                    + validationFileAdapterException.getMessage(), validationFileAdapterException);
```